src/libsyncengine/update_detection/file_system_observer/snapshot/snapshot.cpp

```diff
@@ -466,7 +447,6 @@ bool Snapshot::isOrphan(const NodeId &itemId) {
             // Should not happen
             LOG_WARN(Log::instance()->getLogger(), "Parent ID equals item ID " << nextParentId.c_str());
             assert(false);
-            break;
```

Comment:
Same remark as above: the `break` will not be done for non-debug builds of the application.

Reply:
Done